packages/devextreme/js/__internal/ui/list/m_list.base.ts

```diff
@@ -1127,12 +1195,12 @@ export class ListBase extends CollectionWidget<ListBaseProperties> {
     super._dispose();
   }
 
-  _toggleDisabledState(value): void {
+  _toggleDisabledState(value: boolean): void {
     super._toggleDisabledState(value);
-    this._scrollView.option('disabled', value || !this.option('scrollingEnabled'));
+    this._scrollView?.option('disabled', value || !this.option('scrollingEnabled'));
```

Comment:
this.option('scrollingEnabled') 
-> 
const { scrollingEnabled } = this.option();